clients/src/test/java/org/apache/kafka/clients/consumer/internals/MembershipManagerImplTest.java

```diff
@@ -322,6 +318,7 @@ public void testListenersGetNotifiedOfMemberEpochUpdatesOnlyIfItChanges() {
     private void mockStableMember(MembershipManagerImpl membershipManager) {
         ConsumerGroupHeartbeatResponse heartbeatResponse = createConsumerGroupHeartbeatResponse(new Assignment());
         when(subscriptionState.hasAutoAssignedPartitions()).thenReturn(true);
+        membershipManager.updateAssignment(new HashMap<>());
```

Comment:
I wouldn't expect us to need to manually update the assignment here (because the empty assignment comes from the heartbeat data on ln 319 above). Could you check if/why we need it?